pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -829,10 +829,13 @@ def run(
         Args:
             progress_bar: If True, the progress bar of QuTiP's
                 solver will be shown. If None or False, no text appears.
-            options: If specified, will override
-                SimConfig solver_options. If no `max_step` value is provided,
-                an automatic one is calculated from the `Sequence`'s schedule
-                (half of the shortest duration among pulses and delays).
+            options: Used as arguments for qutip.Options(). If specified, will
+                override SimConfig solver_options. If no `max_step` value is
+                provided, an automatic one is calculated from the `Sequence`'s
+                schedule (half of the shortest duration among pulses and
+                delays).
+                Refer to the QuTiP docs_ for an overview of the parameters.
+.. _docs: https://qutip.org/docs/latest/guide/dynamics/dynamics-options.html
```

Comment:
Maybe using an URL shortener is simpler?

Reply:
You're right - I wasn't sure whether you'd prefer a shortened URL over this.

Reply:
If the original URL had fit, it would indeed have been preferable

Reply:
Unfortunately, it seems it hasn't rendered correctly in the docs: https://pulser--423.org.readthedocs.build/en/423/apidoc/simulation.html#pulser_simulation.simulation.Simulation.run

Reply:
Huh, that's odd. Apparently you need a newline between the end of the text and the start of the links. I've tested it [here](https://livesphinx.herokuapp.com/), so I hope it's fine now.

Reply:
Looks good now :+1:

Reply:
Thanks for your patience!